### ISSUE TITLE
Get rid of $oid

### ISSUE THREAD

**Issue by marcodali** [manyuanrong/deno_mongo#107]
Don't you think it is overcomplicating using

`_id: { $oid: string };`

instead of the simpler

`_id: string` 

We have to make a lot of hacks, validations and wrappers in our code in order to make our API's return database documents.

**Comment by manyuanrong** [comment]
Unable to determine whether the string type is normal text or an ObjectId.

Should use
```ts
import {ObjectId} from "..."
_id: ObjectId("string")
```

**Comment by romaklimenko** [comment]
Is any way to do like:
```
interface MySchema {
  _id: {
    from: string,
    to: string,
    changed: number
  };
  vote: number;
}

const db = client.database("db");
const karmas = db.collection<MySchema>("col");

const insertId = await karmas.insertOne({
  _id: {
      from: "a",
      to: "b",
      changed: 1
  },
  vote: 2
});
```

So far it complains like `Property '$oid' is missing in type '{ from: string; to: string; changed: number; }' but required in type 'ObjectId'.`

Am I missing something or composite _id is not supported?

**Comment by manyuanrong** [comment]
This problem has been implemented in v0.20.0
